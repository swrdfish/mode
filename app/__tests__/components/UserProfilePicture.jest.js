import React from 'react'
import { shallow } from 'enzyme'
import UserProfilePicture from '../../components/UserProfilePicture'
import renderer from 'react-test-renderer';

describe('Component: UserProfilePicture', () => {
    it('should render the profile picture', () => {
        const wrapper = shallow(<UserProfilePicture />);
        expect(wrapper.find(".profile-picture")).toHaveLength(1)
    })

    it('should match previous snapshot', () => {
        const tree = renderer
            .create(<UserProfilePicture />).toJSON();
        expect(tree).toMatchSnapshot();
    })
})