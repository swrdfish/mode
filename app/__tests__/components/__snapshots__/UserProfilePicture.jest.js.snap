// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: UserProfilePicture should match previous snapshot 1`] = `
<div
  className="profile-picture"
/>
`;
